nibabel/cmdline/diff.py

```diff
@@ -125,6 +140,76 @@ def get_data_diff(files):
     return md5sums
 
 
+def get_data_diff(files, max_abs=0, max_rel=0):
+    """Get difference between data
+
+    Parameters
+    ----------
+    files: list of (str or ndarray)
+      If list of strings is provided -- they must be existing file names
```

Comment:
`ndarray` I assume means a data block equivalent to one loaded with `nib.load().get_fdata()` or similar?